blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/formats/AggregatedPayload.java

```diff
@@ -74,6 +75,52 @@ public String toString() {
         return errors;
     }
 
+    public boolean hasDelayedMetrics(long ingestTime) {
+        if ( getDelayTime(ingestTime) > TRACKER_DELAYED_METRICS_MILLIS ) {
```

Comment:
Its probably a matter of preference, but you can just do `return getDelayTime(ingestTime) > TRACKER_DELAYED_METRICS_MILLIS`